e2e-tests/auto-tuning/run

```diff
@@ -7,29 +7,61 @@ test_dir=$(realpath $(dirname $0))
 . ${test_dir}/../functions
 cluster="auto-tuning"
 
+RAM_SIZE=1073741824 # 1Gi
+RDS_MEM_INSTANCE=12582880
 
 create_namespace $namespace
 deploy_operator
 
-spinup_pxc "$cluster" "$test_dir/conf/$cluster.yml"
+spinup_pxc "$cluster" "${test_dir}/conf/${cluster}1.yml"
 
-MYSQL_PASSWORD=$(kubectl_bin get secret my-cluster-secrets -o jsonpath='{.data.root}' | base64 -d)
 AUTO_INNODB_SIZE=$(run_mysql \
                         'SELECT @@innodb_buffer_pool_size;' \
                         "-h $cluster-pxc -uroot -proot_password")
 AUTO_CONNECTIONS=$(run_mysql \
                         'SELECT @@max_connections;' \
                         "-h $cluster-pxc -uroot -proot_password")
 
-kubectl_bin patch pxc "${cluster}" --type merge -p'{"spec": {"pxc": {"configuration": "[mysqld]\ninnodb_buffer_pool_size=150000000\nmax_connections=200\n"}}}'
+if [[ ${AUTO_INNODB_SIZE} != $((${RAM_SIZE} * 75 / 100)) ]]; then
+    echo "innodb_buffer_pool_size is set to ${AUTO_INNODB_SIZE}, which does not correlate with cr.pxc.limits.memory * 0.75"
+    exit 1
+fi
+
+if [[ ${AUTO_CONNECTIONS} != $((${RAM_SIZE} / ${RDS_MEM_INSTANCE})) ]]; then
+    echo "max_connections is set to ${AUTO_CONNECTIONS}, which does not correlate with cr.pxc.limits.memory / ${RDS_MEM_INSTANCE}"
+    exit 1
+fi
+
+kubectl_bin delete pxc --all
+kubectl_bin delete pvc --all
+
+sleep 10
+
+spinup_pxc "$cluster" "${test_dir}/conf/${cluster}2.yml"
+
+AUTO_INNODB_SIZE=$(run_mysql \
+                        'SELECT @@innodb_buffer_pool_size;' \
+                        "-h $cluster-pxc -uroot -proot_password")
+AUTO_CONNECTIONS=$(run_mysql \
+                        'SELECT @@max_connections;' \
+                        "-h $cluster-pxc -uroot -proot_password")
+
+if [[ ${AUTO_INNODB_SIZE} != $((${RAM_SIZE} * 75 / 100)) ]]; then
+    echo "innodb_buffer_pool_size is set to ${AUTO_INNODB_SIZE}, which does not correlate with cr.pxc.requests.memory * 0.75"
+    exit 1
+fi
+
+if [[ ${AUTO_CONNECTIONS} != $((${RAM_SIZE} / ${RDS_MEM_INSTANCE})) ]]; then
+    echo "max_connections is set to ${AUTO_CONNECTIONS}, which does not correlate with cr.pxc.requests.memory / ${RDS_MEM_INSTANCE}"
+    exit 1
+fi
+
+kubectl_bin delete pxc --all
+kubectl_bin delete pvc --all
 
+sleep 10
 
-wait_cluster_consistency "${cluster}" 3
-sleep 20
-wait_cluster_consistency "${cluster}" 3
-sleep 20
-wait_cluster_consistency "${cluster}" 3
-sleep 20
+spinup_pxc "$cluster" "${test_dir}/conf/${cluster}3.yml"
```

Comment:
can we just apply "${test_dir}/conf/${cluster}3.yml" without deleting cluster?

Reply:
done